src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -241,6 +413,11 @@ ucs_status_t uct_rc_mlx5_iface_common_init(uct_rc_mlx5_iface_common_t *iface,
         return status;
     }
 
+    status = uct_mlx5_iface_init_dm(iface, rc_iface, common_config);
+    if (status != UCS_OK) {
+        return status;
```

Comment:
you would need to release the dm later if STATS is not allocated

Reply:
released